Automattic/pocket-casts-android
# Fix font scaling

## Description

While doing code review I noticed that font scaling wasn't applied correctly to all of our texts. This PR applied it to all font sizes, line heights, and letter spacing in our compose elements.

## Testing Instructions

1. Increase font size on your devices.
2. Notice that a screen like profile tab scales correctly.
3. Start PB24.
4. Notice that the fonts aren't scaled.

## Checklist
 
#### I have tested any UI changes...
